project_legal_management/views/project_tree.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl) -->
+<odoo>
+    <record id="view_project" model="ir.ui.view">
+        <field name="name">view.project</field>
```

Comment:
Add here something to identify the module on view hierarchy, something like "view.project.legal.management"